### ISSUE TITLE
when official documentation of the iTunes Connect JSON API will release ??

### ISSUE THREAD

**Issue by Aravindios** [fastlane/itc-api-docs#29]
when official documentation of the iTunes Connect JSON API will release ??
 apple said iTC API will release in summer ???

**Comment by KrauseFx** [comment]
Yes, this is not related to this repo though, but good point, we should update the README to reflect that. @joshdholtz is that something you could take on?
